index.js

```diff
@@ -419,14 +420,18 @@ export default class Carousel extends Component {
             index = 0;
             fireCallback = false;
         }
+        else if(index === this.state.oldItemIndex) {
```

Comment:
@rgabs In order to keep current code formatting, could you move your condition to the previous line and add a space before the opening parenthesis?

=> `} else if (index === this.state.oldItemIndex) {`